src/client/pages/Varen/useVarenListPage.hook.ts

```diff
@@ -0,0 +1,37 @@
+import {
+  varenMeerInformatieLink,
+  tableConfig,
+  exploitatieVergunningAanvragen,
+} from './Varen-thema-config';
+import { VarenVergunningFrontend } from '../../../server/services/varen/config-and-types';
+import { isError, isLoading } from '../../../universal/helpers/api';
+import { addLinkElementToProperty } from '../../components/Table/TableV2';
+import { useAppStateGetter } from '../../hooks/useAppState';
+
+export type VarenVergunningKind = 'open' | 'afgehandeld' | 'overgedragen';
```

Comment:
Op andere plekken definiëren we dit in de config file:

```
export type ListPageParamKey = keyof typeof listPageParamKind;
export type ListPageParamKind = (typeof listPageParamKind)[ListPageParamKey];
```
Dan zie je ook dat `'open' 'afgehandeld ' en 'overgedragen'` niet bestaan. Ik zie wel `"lopende-aanvragen"` en `"afgehandelde-aanvragen"`